.gitignore

```diff
@@ -5,3 +5,5 @@
 
 /_output
 /vendor
+/tests/smoke/_output
```

Comment:
why is this needed? all output should go in _output

Reply:
when you run int-tests, the logs are outputted to this directory.

Reply:
@dangula why is that? if they don't go to _output they won't be copied to the s3 bucket.

Reply:
that is not needed,  all output is going to _output dir and I see then being coped to s3 bucket. 
when tests are run from IDE or using go test  from /tests/smoke dir a temp _output folder is created to stash rook logs(in case of failures)

Reply:
i did run the test locally using go test. i will remove this then